kubernetes/clickhouse/templates/clickhouseKeeper.yaml

```
@@ -0,0 +1,55 @@
+apiVersion: "clickhouse-keeper.altinity.com/v1"
+kind: "ClickHouseKeeperInstallation"
+metadata:
+  name: {{ include "common.names.fullname" . }}-keeper
+  labels:
+    {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: {{ .Chart.Name }}-keeper
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)